src/components/graph.tsx

```diff
@@ -89,10 +91,61 @@ export class Graph extends React.Component<GraphProps, GraphState> {
             yAxisFix: Format.getAxisFix('y', this.props.yMax - this.props.yMin, this.props.height)
         };
 
+        this.getPlotterType = this.getPlotterType.bind(this);
+        this.dataForMultiBar = this.dataForMultiBar.bind(this);
+        this.useMultiBar = this.useMultiBar.bind(this);
+        this.makeDygraph = this.makeDygraph.bind(this);
         this.dyUpdateProps = ["width", "height", "xMin", "xMax", "yMin", "yMax",
                                 "valuePrecision", "xLabel", "yLabel", "predictionState"];
     }
 
+    useMultiBar() {
```

Comment:
Could this be simplified to something like the below?

```suggestion
useMultiBar() {
  const { displayType, useAuthoredData, usePrediction } = this.props;
  return displayType === "bar" && (useAuthoredData || usePrediction);
}
```